lib/LINZ/BdeUpload.pm

```diff
@@ -1021,10 +1021,21 @@ sub LoadFile
             if $checktime;
 
         # Determine which columns are to be copied
-        my $columns = join("|",$reader->fields);
+        my $fields = join("|",$reader->fields);
+        if ($fields eq '') {
+            die "BDE file has no fields defined";
+        }
+        DEBUG("BDE Fields: $fields");
+
+        my $columns = $db->selectValidColumns($tablename,$fields);
+        DEBUG("DB Columns: $columns");
```

Comment:
Small comment (up to you), but you should be consistent when you log the fields and columns variables. One is before and one is after the `eq ''` test

Reply:
Thanks for the comment. Tweaked with 9820eec (and now reviews were invalidated so could you please approve this for me again ?)